widgetry/src/widgets/button.rs

```diff
@@ -195,8 +195,64 @@ impl Btn {
         }
     }
 
-    pub fn custom(normal: GeomBatch, hovered: GeomBatch, hitbox: Polygon) -> BtnBuilder {
-        BtnBuilder::Custom(normal, hovered, hitbox, None)
+    pub fn pop_up<I: Into<String>>(ctx: &EventCtx, label: Option<I>) -> BtnBuilder {
+        let mut icon_batch = GeomBatch::new();
+        let icon_container = Polygon::rectangle(20.0, 30.0);
```

Comment:
Is there a better way to "add padding" around an svg?

`icon_batch.batch().container().padding(10)` didn't seem to do anything.

Reply:
Hmm, I'd expect that to work. I can try some variations later and figure out what's going wrong. Would you use it here instead of the invisible `icon_container`?